plugins/org.jkiss.dbeaver.ui.dashboard/src/org/jkiss/dbeaver/ui/dashboard/registry/DashboardDescriptor.java

```diff
@@ -104,7 +104,25 @@ public DataSourceMapping(String dataSourceProvider, String driverId, String driv
 
         boolean matches(String providerId, String checkDriverId, String checkDriverClass) {
             if (this.dataSourceProvider != null && !this.dataSourceProvider.equals(providerId)) {
-                return false;
+                // Providers not equals, ok, but we can check children providers then
```

Comment:
This comment is a bit goofy.

```suggestion
                // Try finding a matching child provider
```